include/nil/blueprint/blueprint/plonk/assignment_proxy.hpp

```diff
@@ -92,9 +96,13 @@ namespace nil {
             }
 
             value_type selector(std::size_t selector_index, std::uint32_t row_index) const override {
-                if (check && used_rows.find(row_index) == used_rows.end()) {
-                    std::cout << id << ": Not found selector " << selector_index << " on row " << row_index << std::endl;
-                    BLUEPRINT_ASSERT(false);
+                if (check) {
+                    const auto lookup_selector_cols = assignment_ptr->get_lookup_selector_cols();
```

Comment:
This checks only `lookup_selector_cols`, is this the intended behavior?

Reply:
not only, in case of regular selector: `lookup_selector_cols.find(selector_index) == lookup_selector_cols.end()`, so we'll check second part of condition `used_rows.find(row_index) == used_rows.end()` which means check if the regular selector on the private row for current prover.